src/requisition-order-create/order-create-tab.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState, useMemo } from 'react';
+import Tippy from '@tippyjs/react';
+
+import EditableTable from '../react-components/table/editable-table';
+import OrderCreateRequisitionInfo from './order-create-requisition-info';
+
+import { SearchSelect } from './search-select';
+import { getUpdatedOrder } from './order-create-table-helper-functions';
+import { orderTableColumns } from './order-create.constant';
+import { validateOrderItem } from './order-create-validation-helper-functions';
+
+
+const OrderCreateTab = ({ passedOrder, orderableOptions, updateOrderArray, showValidationErrors }) => {
+    const [order, setOrder] = useState({ orderLineItems: [], ...passedOrder });
+    const [selectedOrderable, selectOrderable] = useState('');
```

Comment:
Adhere to the convention of using 'set' as a prefix for setter methods.
`
const [selectedOrderable, setSelectedOrderable] = useState(' ');
`